lang/java/reef-applications/reef-vortex/src/test/java/org/apache/reef/vortex/driver/TestUtil.java

```diff
@@ -45,10 +49,14 @@
   private final Executor executor = Executors.newFixedThreadPool(5);
   private final VortexMaster vortexMaster = mock(VortexMaster.class);
 
+  public VortexWorkerManager newWorker() {
```

Comment:
I think it'd be good to add a javadoc here: actually it would be the same as the previous one :)